packages/databricks-sdk-js/src/config/Config.ts

```diff
@@ -258,7 +258,10 @@ export class Config {
      */
     public isAzure(): boolean {
         return (
-            (!!this.host && this.host.endsWith(".azuredatabricks.net")) ||
+            (!!this.host &&
+                !!this.host.match(
+                    /(\.databricks\.azure\.us|\.databricks\.azure\.cn|\.azuredatabricks\.net)$/
```

Comment:
`!!` to boolify the result?

The regexp could be collapsed with `(us|cn)`.

Reply:
Yes `!!` is a JS idiom to convert a truthy of falsy value to a proper boolean